notebooks/src/3-create-stac-item-with-extension.ipynb

```diff
@@ -321,45 +1734,45 @@
        "[<Item id=local-image-eo>]"
       ]
      },
-     "execution_count": 13,
+     "execution_count": 19,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "catalog.add_item(eo_item)\n",
+    "catalog.add_item(item)\n",
     "list(catalog.get_items())"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
+   "execution_count": 20,
    "metadata": {},
    "outputs": [],
    "source": [
-    "catalog.normalize_and_save(root_href=os.path.join(tmp_dir.name, 'stac-eo'), \n",
+    "catalog.normalize_and_save(root_href=os.path.join(tmp_dir.name, 'stac-extension'), \n",
     "                           catalog_type=pystac.CatalogType.SELF_CONTAINED)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, if we read the catalog from the filesystem, PySTAC recognizes that the item implements eo and to use its functionality, e.g. getting the bands off the asset:"
+    "Now, if we read the catalog from the filesystem, PySTAC recognizes that the item implements eo and other extensions and to use its functionality, e.g. getting the bands off the asset:"
```

Comment:
Reword: "Now, if we read the catalog from the filesystem, PySTAC recognizes that the item implements eo and other extensions and to use its functionality, e.g. getting the bands off the asset:"

to "Now, if we read the catalog from the file system, PySTAC recognizes that the item implements some extensions and uses their functionality."

And then right before the "print(eo_on_asset.bands)" code block, add the following line: "Below, we can see the bands of the asset from the information in the eo extension:"